specs/windows/wmi_filter_consumer_binding.table

```diff
@@ -0,0 +1,12 @@
+table_name("wmi_filter_consumer_binding")
+description("Lists the relationship between event consumers and filters.")
+schema([
+    Column("consumer", TEXT, "The parent registry hive where the key and value exist"),
+    Column("filter", TEXT, "The parent registry hive where the key and value exist"),
+    Column("__class", TEXT, "The parent registry hive where the key and value exist"),
```

Comment:
Not really, it's just what windows calls it. Run this and you'll see

get-wmiobject -namespace root\subscription -class __FilterToConsumerBinding